playground/Makefile

```
@@ -0,0 +1,18 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/ai-lab-recipes/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
+	curl -s -S -L -f https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -z $@ -o $@.tmp && mv -f $@.tmp $@ 2>/dev/null || rm -f $@.tmp $@
+
+.PHONY: download
+download:
```

```suggestion
.PHONY: install
install:
```
Does it make sense to rename this "install" and add "download" above for the model?

---

Reverted